pyirf/interpolation/moment_morph_interpolator.py

```diff
@@ -238,10 +238,6 @@ def moment_morph_estimation(bin_edges, bin_contents, coefficients):
     # Reset interpolation resolts for those templates with partially zero entries from above to 0
     f_new[zero_templates] = np.zeros(len(bin_mids))
 
-    # If used normally, all values are stricktly positive, if used for extrapolation, some values can
-    # become slightly negative as the mean/std estimation is not exact.
-    f_new[f_new < 0] = 0
```

Comment:
What changed?

Reply:
Ah, moved to the extrapolation only...

Reply:
Yes, since for interpolation all coefficients are strictly positive and with those positive coefficients strictly positive bin-entries are overlaid, this is only needed for extrapolation where coefficients are negative.